src/main/java/core/basesyntax/Plane.java

```diff
@@ -1,5 +1,72 @@
 package core.basesyntax;
 
 public class Plane {
+    private String model;
+    private String color;
+    private int flightRange;
+    private int maxSpeed;
+    private int passengerCapacity;
 
+    private Plane(PlaneBuilder planeBuilder) {
+        this.model = planeBuilder.model;
+        this.color = planeBuilder.color;
+        this.flightRange = planeBuilder.flightRange;
+        this.maxSpeed = planeBuilder.maxSpeed;
+        this.passengerCapacity = planeBuilder.passengerCapacity;
+    }
+
+    @Override
+    public String toString() {
+        return "Plane{"
+                + "model='"
```

Comment:
```suggestion
                + "model='" + model + '\''
```
Let's inline these statements